pyvista/core/filters/data_set.py

```diff
@@ -1414,7 +1414,13 @@ def extract_all_edges(self, use_all_points=False, progress_bar=False):
                     'This version of VTK does not support `use_all_points=True`. '
                     'VTK v9.1 or newer is required.'
                 )
+        if _vtk.VTK9:
+            # vtkExtractEdges improperly uses INFO for debugging messages
+            _vtk.vtkLogger.SetStderrVerbosity(_vtk.vtkLogger.VERBOSITY_OFF)
```

Comment:
Non blocker: has this been addressed upstream?

Reply:
It's a least known about... I will open a proper issue if not tracked/addressed yet

Reply:
Tracked in https://gitlab.kitware.com/vtk/vtk/-/issues/18785